src/history.cpp

```diff
@@ -1453,7 +1453,10 @@ bool history_t::save_internal_via_rewrite()
                 if (wstat(new_name, &sbuf) >= 0)
                 {
                    /* Success */
-                    fchown(out_fd, sbuf.st_uid, sbuf.st_gid);
+                    if (0 > fchown(out_fd, sbuf.st_uid, sbuf.st_gid))
+                    {
+                        debug(2, L"Error when changing ownership of history file");
```

Comment:
As in
```
debug(1, L"Error when changing ownership of history file: %s", safe_strerror(errno));
```
?

I don't see another example of outputting ```errno``` elsewhere in the code.  I do see examples of ```perror```, but not used in conjunction with ```debug```. 
Two statements?  The ```debug``` and a ```wperror```?